projects/1.20/assets/mob-catcher-fabric/mob_catcher/lang/zh_cn.json

```
@@ -0,0 +1,19 @@
+{
+  "item.mob_catcher.diamond_mob_catcher": "钻石生物抓捕绳",
+  "item.mob_catcher.diamond_mob_catcher.line1": "抓捕并储存所有被动生物",
+  "item.mob_catcher.diamond_mob_catcher.line2": "不对玩家生效",
+  "item.mob_catcher.diamond_mob_catcher.line3": "装有：空",
+  "item.mob_catcher.diamond_mob_catcher.line4": "装有：%s",
+  "item.mob_catcher.diamond_mob_catcher.line5": "左击抓捕",
+  "item.mob_catcher.diamond_mob_catcher.line6": "右击释放",
+
+  "item.mob_catcher.netherite_mob_catcher": "下界生物抓捕绳",
+  "item.mob_catcher.netherite_mob_catcher.line1": "抓捕并储存所有被动和敌对生物",
+  "item.mob_catcher.netherite_mob_catcher.line2": "不对玩家和凋灵生效",
+  "item.mob_catcher.netherite_mob_catcher.line3": "装有：空",
+  "item.mob_catcher.netherite_mob_catcher.line4": "装有：%s",
+  "item.mob_catcher.netherite_mob_catcher.line5": "左击抓捕",
+  "item.mob_catcher.netherite_mob_catcher.line6": "右击释放",
+
+  "itemGroup.mob_catcher": "生物抓捕栓绳"
```

```suggestion
  "itemGroup.mob_catcher": "生物抓捕绳"
```
物品名和模组名相同且应该指代的是同一类物品，注意统一